test/holes_test.py

```diff
@@ -30,12 +30,10 @@ def test_holes(self):
     def test_holes_more(self):
         f = 10
         index = AnnoyIndex(f)
-        valid_indices = set()
-        for i in range(1000):
-            i2 = int(i*2**-0.5) # leave holes every few items
-            valid_indices.add(i2)
+        valid_indices = range(0, 2000, 2) # leave holes
```

Comment:
do you mind replacing this with `random.sample(range(2000), 1000)` to make it a bit uneven?

Reply:
sure thing